test/float.cc

```diff
@@ -0,0 +1,69 @@
+#include "criterion/parameterized.h"
+#include "criterion/options.h"
+#include "criterion/theories.h"
+#include "criterion/float.h"
+#include <cmath>
+#include <cfloat>
+#include <iostream>
+#include <unistd.h>
```

Comment:
Actually unistd.h isn't used at all, it's just a forgotten artifact that I'll most probably remove later.